dev_environment/README.md

```diff
@@ -65,3 +67,19 @@ Here is a visual overview:
 * To see the eBPF program metrics, browse to `http://localhost:33000` on the
   host and login to Grafana with the default username and password of `admin`.
   After logging in you will be able to view the preconfigured dashboards.
+
+# Trying out L3AF on an Azure VM (Ubuntu 20.04.4 LTS)
+[setup_dev_env_linux_vm.md](setup_dev_env_linux_vm.md) illustrates host prerequisites and installation instructions for trying out L3AF on an Azure VM running an Ubuntu 20.04.4 LTS server.
+
+## Trying out your L3AFD Server
```

Comment:
This section is almost the same as what I have above minus the screenshots.  If we want to combine them that would be better than having two different sections.  We can also just drop this entirely and keep what is above.

Reply:
I combined them and showed the screenshots at the right place.